bots/images/scripts/rhel.setup

```diff
@@ -170,6 +173,7 @@ if [ "$IMAGE" = "rhel-x" ]; then
     COCKPIT_DEPS="${COCKPIT_DEPS/atomic /}"
     # docker is not a rhel-x thing (https://bugzilla.redhat.com/show_bug.cgi?id=1602951)
     COCKPIT_DEPS="${COCKPIT_DEPS/docker /}"
+    COCKPIT_DEPS="${COCKPIT_DEPS/libvirt-daemon-kvm /}"
```

Comment:
I think you mean to *add* the package here, not remove it?

Reply:
Right and also added libvirt-daemon-config-network which brings default network.